### PR TITLE
Adding information on how to install FEniCS

### LINKED ISSUE #82: Missing instructions on how to install FEniCS
In the [FEniCS adapter documentation](https://precice.org/adapter-fenics.html), we currently have instructions on how to install the adapter.

In other adapter, for example for [OpenFOAM](https://precice.org/adapter-openfoam-support.html) and for [CalculiX](https://precice.org/adapter-calculix-get-calculix.html), we also have some instructions on how to install the solver itself first. It would be useful to have some short instructions here as well with the preferred or most common way to install FEniCS (as well as which version and any other questions that the user may have).

One would argue "it is not our job to document the solvers", but it is actually to our strong favor. New users often know/have one of the solvers, but they know nothing about the rest of the solvers they need to run tutorials. Even further, many of our users are students that do not have much experience with simulation software and installing software on Linux anyway. Giving some first guidance definitely helps users start with preCICE.

@IshaanDesai @BenjaminRodenberg would you maybe like to add something here? I could also prepare something, but you are probable the right people for that.

### PR DESCRIPTION
**Pull request by IshaanDesai**
This PR adds a section to the FEniCS-preCICE adapter page providing information on how to install FEniCS. Closes https://github.com/precice/precice.github.io/issues/82

### CODE REVIEW

File: pages/docs/adapters/fenics/adapter-fenics.md [hunk 1]
```diff
@@ -5,15 +5,26 @@ keywords: adapter, fenics
...
 
+## How to get FEniCS
+
+You can install FEniCS on your system by several ways as mentioned on [fenicsproject.org](https://fenicsproject.org/download/). The simplest way to install FEniCS on Ubuntu is using `apt-get`:
+
+```bash
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:fenics-packages/fenics
+sudo apt-get update
+sudo apt-get install fenics
```
Comment (MakisH) [inline pages/docs/adapters/fenics/adapter-fenics.md]: In (most of) the rest of the documentation, we use `apt install`, `apt update`, and `apt install`. The `apt-get` is an older command and I think there are no actively supported distributions still not supporting `apt`.

Side note: we should still use `apt-get` in scripts, as `apt` itselfs says it is not meant to be used in scripts at the moment (reports so when trying to run in a script).

```suggestion
sudo apt install software-properties-common
sudo add-apt-repository ppa:fenics-packages/fenics
sudo apt update
sudo apt install fenics
```

https://askubuntu.com/a/446484/142834

File: pages/docs/adapters/fenics/adapter-fenics.md [hunk 2]
```diff
@@ -5,15 +5,26 @@ keywords: adapter, fenics
...
 ---
 
-## What is FEniCS ?
+## What is FEniCS
 
 From the FEniCS website: _FEniCS is a popular open-source (LGPLv3) computing platform for solving partial differential equations (PDEs). FEniCS enables users to quickly translate scientific models into efficient finite element code. With the high-level Python and C++ interfaces to FEniCS, it is easy to get started, but FEniCS offers also powerful capabilities for more experienced programmers. FEniCS runs on a multitude of platforms ranging from laptops to high-performance clusters._ More details can be found at [fenicsproject.org](https://fenicsproject.org/).
 
+## How to get FEniCS
+
+You can install FEniCS on your system by several ways as mentioned on [fenicsproject.org](https://fenicsproject.org/download/). The simplest way to install FEniCS on Ubuntu is using `apt-get`:
```
Comment (MakisH) [inline pages/docs/adapters/fenics/adapter-fenics.md]:
```suggestion
You can install FEniCS on your system by several ways as mentioned on [fenicsproject.org](https://fenicsproject.org/download/). The simplest way to install FEniCS on Ubuntu is using the official PPA repository of FEniCS:
```

**Review by MakisH — APPROVED** [review]
Looks good already, thanks!

I currently get segmentation faults immediately when trying to run any FEniCS tutorial, but I guess this is not something we need to wait for here.

In the [vm](https://github.com/precice/vm/blob/main/provisioning/install-fenics.sh), we use the `--no-install-recommends` option. I don't remember the details now, but I think the PPA repository also contains some alternative versions of some dependencies that somehow caused issues on Ubuntu 20.04. This does not fix my issue.

**Comment by IshaanDesai** [comment]
> I currently get segmentation faults immediately when trying to run any FEniCS tutorial, but I guess this is not something we need to wait for here.

Do you get a segmentation fault for any of our tutorial cases or some other tutorials provided by FEniCS? If it is the former case then we should look into via an issue in the tutorials repo.

**Comment by MakisH** [comment]
> Do you get a segmentation fault for any of our tutorial cases or some other tutorials provided by FEniCS? If it is the former case then we should look into via an issue in the tutorials repo.

Not sure about FEniCS-only tutorials. What could I run? We could also debug this together offline.

**Comment by IshaanDesai** [comment]
> Not sure about FEniCS-only tutorials. What could I run? We could also debug this together offline.

Okay no I just misunderstood you. I understand now that you ran one of the preCICE tutorials with FEniCS as a participant. If your FEniCS installation is suddenly broken then please have a look at the [troubleshooting section](https://github.com/precice/fenics-adapter#troubleshooting). Otherwise yes lets debug this together offline sometime.

**Pull request merged**